openmdao/visualization/meta_model_viewer/tests/test_unstruct.py

```diff
@@ -217,5 +251,76 @@ def test_not_top_level_prob(self):
+
+        adjusted_points = MetaModelVisualization(self.mm)
+        adjusted_points.input_point_list = [0.04203304, 2.043553874897959, 0.02435233]
+        adjusted_points.dist_range = 0.75
+
+        new_points = adjusted_points._unstructured_training_points(compute_distance=True, source='bottom')
+
+        # Make sure that arrays equal each other to the 6th decimal place
+        assert_almost_equal(known_points, new_points, decimal=4)
+
+    def test_alpha_transparency(self):
+
+        adjusted_points = MetaModelVisualization(self.mm)
+        adjusted_points.input_point_list = [0.04203304, 2.043553874897959, 0.02435233]
+        adjusted_points.dist_range = 0.75
+
+        known_points_right = np.array([0.86567999, 0.78348615, 0.63977714, 0.62941009, 0.62587479,
+                                    0.56152768, 0.50987912, 0.49376081, 0.48388929, 0.46630135,
+                                    0.46401112, 0.4598095 , 0.41160183, 0.39350238, 0.39026133,
+                                    0.38265734, 0.38192589, 0.36878817, 0.34231522, 0.32336566,
+                                    0.30383293, 0.29137251, 0.28730615, 0.28575177, 0.26159435,
+                                    0.26053391, 0.2252101 , 0.18561353, 0.17247848, 0.13633159,
+                                    0.10337788, 0.10021363, 0.08660427, 0.08069673, 0.06691631,
+                                    0.0547412 , 0.04688812, 0.04670322, 0.01284644, 0.00901997,
+                                    0.00842558, 0.00386698])
+
+        known_points_bottom = np.array([0.765526225, 0.738928771, 0.726005596, 0.715400685,
+                                        0.649085936, 0.640394534, 0.628033739, 0.613040224,
+                                        0.583063429, 0.439487175, 0.368150584, 0.343219846,
+                                        0.323457648, 0.299935318, 0.252810468, 0.244806901,
+                                        0.243472059, 0.236658572, 0.233785726, 0.222517297,
+                                        0.208587812, 0.199369605, 0.195640700, 0.180272582,
+                                        0.174355523, 0.152993107, 0.128729120, 0.128385050,
+                                        0.128254288, 0.105788052, 0.101550328, 0.0844650938,
+                                        0.0639579453, 0.0334477759, 0.0197405899, 0.000640993538])
+
+        right_points = adjusted_points._unstructured_training_points(compute_distance=True, source='right')
+        right_plot = adjusted_points._right_plot()
+        right_transparency = adjusted_points.right_alphas
+
+        bottom_points = adjusted_points._unstructured_training_points(compute_distance=True, source='bottom')
+        bottom_plot = adjusted_points._bottom_plot()
+        bottom_transparency = adjusted_points.bottom_alphas
+
+        # Make sure that arrays equal each other to the 6th decimal place
+        assert_almost_equal(known_points_right, right_transparency, decimal=5)
```

Comment:
could also use `np.testing.assert_allclose` to check values.  It allows you to specify relative or absolute tolerance.